staging/src/k8s.io/api/core/v1/well_known_labels.go

```diff
@@ -17,9 +17,12 @@ limitations under the License.
 package v1
 
 const (
-	LabelHostname          = "kubernetes.io/hostname"
-	LabelZoneFailureDomain = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
These are defined here but couldn't find anything setting these labels on any resource so i think it's okay to remove

Reply:
redefining these constants to different values is not good. see the approach taken for the OS/Arch label constants in https://github.com/kubernetes/kubernetes/pull/73333/files#diff-e567833c41b619e81b80227c8b8e1c43

it would be better to delete these constants than to redefine their values